package/CHANGELOG

```diff
@@ -58,6 +58,8 @@ Fixes
     Universe (PR #2893)
   * ensure that unistd.h is included on macOS when compiling ENCORE's spe.c
     (Issue #2934)
+  * Fix tests for analysis.bat that could fail when run in parallel and that
```

Comment:
I guess technically it should be newest first, but I've seen plenty of cases where we've not followed this so it's a bit late to be pedantic now 🤷‍♂️

Reply:
Oops... I have been doing it wrong for a while 😅